addons/main/script_mod.hpp

```diff
@@ -10,7 +10,7 @@
 #define VERSION_AR  MAJOR,MINOR,PATCHLVL,BUILD
 
 // MINIMAL required version for the Mod. Components can specify others..
-#define REQUIRED_VERSION 2.02
+#define REQUIRED_VERSION 2.04
```

Comment:
This is changed in #587. Doesn't it make sense for it there to be removed?

Reply:
Removed from this PR after merging master into this PR's branch.